cross_compile/Dockerfile_ros

```diff
@@ -31,32 +34,38 @@ RUN echo 'en_US.UTF-8 UTF-8' >> /etc/locale.gen && \
 ENV LANG en_US.UTF-8
 ENV LC_ALL C.UTF-8
 
-# Add the ros2 apt repo
+# Add the ros apt repo
 RUN apt-get update && apt-get install -y \
         curl \
         gnupg2 \
         lsb-release \
     && rm -rf /var/lib/apt/lists/*
 RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
-RUN sh -c 'echo "deb http://packages.ros.org/ros2/ubuntu `lsb_release -cs` main" \
-    > /etc/apt/sources.list.d/ros2-latest.list'
 
-# ROS2 dependencies
+RUN if [[ "${ROS_VERSION}" == "ROS2" ]]; then \
+    echo "deb http://packages.ros.org/ros2/ubuntu `lsb_release -cs` main" \
```

Comment:
We should pass $ROS_VERSION = "ros" or "ros2", then this can skip the conditional and be

```
RUN echo "deb http://packages.ros.org/${ROS_VERSION}/ubuntu `lsb_release -cs` main" > /etc/apt/sources.list.d/ros-latest.dist
```

Reply:
Ah - good idea! Will change it.